styles.css

```diff
@@ -0,0 +1,77 @@
+body {
+  margin: -1px;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+header {
+  display: flex;
+  background: #3c3a39;
+  color: #fff;
+  justify-content: space-around;
+  height: 48px;
+  align-items: center;
+  font-size: 18px;
+}
```

Comment:
- [ ] It would be nice to put a `margin-right` and `margin-left` of `24px or 1.5rem` which is the same measurement used in the _**Figma**_ template you chose. This can be done before applying flexbox in the header section. 
- [ ] Kindly use the `Roboto` font-family for the `Logo`. One of the project requirements is that your design should use the same fonts, colors, and margins used in the Figma template you chose. 🙏

Reply:
I have a problem with the second suggested change, linters sends an error when I try to use the original font from Figma